programs/mango-v4/src/accounts_ix/openbook_v2_cancel_order.rs

```diff
@@ -0,0 +1,55 @@
+use anchor_lang::prelude::*;
+
+use crate::error::*;
+use crate::state::*;
+use openbook_v2::{
+    program::OpenbookV2,
+    state::{Market, OpenOrdersAccount},
+};
+
+#[derive(Accounts)]
+pub struct OpenbookV2CancelOrder<'info> {
+    #[account(
+        constraint = group.load()?.is_ix_enabled(IxGate::OpenbookV2CancelOrder) @ MangoError::IxIsDisabled,
+    )]
+    pub group: AccountLoader<'info, Group>,
+
+    #[account(
+        mut,
+        has_one = group,
+        constraint = account.load()?.is_operational() @ MangoError::AccountIsFrozen
+        // owner is checked at #1
```

Comment:
All accounts structs: `// authority is checked at #1`

Reply:
replaced all remaining owners.